indico/migrations/versions/20230905_1200_e2b69fe5155d_add_user_data_export_table.py

```diff
@@ -0,0 +1,48 @@
+"""Add data_export_requests table
+
+Revision ID: e2b69fe5155d
+Revises: 0c44046dc1be
+Create Date: 2023-05-08 15:37:24.134940
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import ARRAY
+
+from indico.core.db.sqlalchemy import PyIntEnum, UTCDateTime
+from indico.modules.users.models.export import DataExportOptions, DataExportRequestState
```

Comment:
don't import enums, redefine them in the model (see some earlier revisions, searching in them for `class _` will probably yield good results) - otherwise the revision will break if the enum gets changed in the future